osu.Game/Online/Leaderboards/LeaderboardScoreTooltip.cs

```diff
@@ -101,7 +111,7 @@ public void SetContent(ScoreInfo score)
 
             displayedScore = score;
 
-            timestampLabel.Text = $"Played on {score.Date.ToLocalTime():d MMMM yyyy HH:mm}";
+            prefer24HourTime.BindValueChanged(_ => updateDisplay(), true);
```

Comment:
This is done in the wrong place. It needs to be in `LoadComplete` and a single call to `updateDisplay()` needs to be in this method. The way you have it, ever time a new score is displayed in the tooltip the will be bound another time, leading to performance issues.

Also `updateDisplay` should be renamed to `updateTimestampLabel`, and the method should be moved below this one.

Reply:
That makes sense, i've fixed it!